Automattic/wp-calypso
# Domains: Don't display cancel button if pending registry

Tell users that at pending_registry status a transfer in is not possible to cancel anymore from our side. Only from the losing registrar side.

Test:
- Start a transfer in
- Hack backend to return `pending_registry` status
- Open the domain in domain management
- Make sure no cancel button
- Check the message saying you cannot cancel the transfer in
